api/PerformanceNavigationTiming.json

```diff
@@ -300,6 +300,39 @@
           }
         }
       },
+      "notRestoredReasons": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceNavigationTiming/notRestoredReasons",
+          "support": {
+            "chrome": {
+              "version_added": "116"
```

Comment:
Turns out this did not ship in 116 after all. It's likely to be 117 but might slip to 118. Keep an eye on https://chromestatus.com/feature/5684908759449600 (which at the time of writing incorrectly states it did ship in 116).

Reply:
Thx @tunetheweb. I've updated the version to 117 for now, and will look out for other changes.

Reply:
@tunetheweb The ChromeStatus entry said it shipped in 117. Did it? ;-)

Reply:
Nope! Got them to move forward to 118 but wouldn't surprise me if that slips again. There's some discussion on the [HTML standard repo PR](https://github.com/whatwg/html/pull/9360) so looking to get that all ironed out before enabling by default.